velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -407,12 +407,12 @@ TEST_F(DateTimeFunctionsTest, fromUnixtime) {
   EXPECT_EQ(Timestamp(-1, 9000), fromUnixtime(-0.999991));
   EXPECT_EQ(Timestamp(4000000000, 0), fromUnixtime(4000000000));
   EXPECT_EQ(
-      Timestamp(9'223'372'036'854'775, 807'000'000), fromUnixtime(3.87111e+37));
```

Comment:
The original test seems correct to me. See https://github.com/prestodb/presto/issues/21025#issuecomment-1748556485